cypress/e2e/group2/myworkflows.ts

```diff
@@ -542,6 +542,7 @@ describe('Dockstore my workflows part 3', () => {
       cy.wait(1000);
       cy.get('#2-register-workflow-option').click();
       cy.contains('button', 'Next').click();
+      cy.wait(2000); // Give animation a chance to kick in.
```

Comment:
Here and in mytools.ts, this seems to fix flaky tests in registration dialog, so far.

Reply:
yay

Reply:
Ohh just saw this, do you have a sense of how reliable this fix is? Perhaps I should see how this fix plays out before merging https://github.com/dockstore/dockstore-ui2/pull/1991?

Reply:
`wait`s always seem like a stopgap solution. I'd rather go with yours. Mine has worked 2 out of 2 times, less conclusive than your testing.